### PR TITLE
CRM457-1474: Amend NSM home/landing/root page to be your claims

### PR DESCRIPTION
**Pull request by jsugarman**
## Description of change
Amend NSM landing/home/root page to be your claims

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1474)

Refactor your, all and closed claims routes and
root to your claims.

## Notes for reviewer

This refactor follows a similar pattern to prior authority for
consistency and because it is generally clearer than having
multiple controllers, despite that being more "Restful".

### CODE REVIEW

File: app/views/layouts/_primary_navigation.html.erb [hunk 1]
```diff
@@ -1,16 +1,18 @@
+<% current ||= :your %>
```
Comment (jsugarman) [inline app/views/layouts/_primary_navigation.html.erb:1]: remove

File: bin/rake [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
+load File.expand_path("spring", __dir__)
```
Comment (jsugarman) [inline bin/rake:3]: sneaked in from what i though was only a local change attempt to speed up rspec and rubocop - can remove

File: bin/spring [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+if !defined?(Spring) && [nil, "development", "test"].include?(ENV["RAILS_ENV"])
+  require "bundler"
+
+  Bundler.locked_gems.specs.find { |spec| spec.name == "spring" }&.tap do |spring|
+    Gem.use_paths Gem.dir, Bundler.bundle_path.to_s, *Gem.path
+    gem "spring", spring.version
+    require "spring/binstub"
+  end
+end
```
Comment (jsugarman) [inline bin/spring:14]: sneaked in from what i though was only a local change attempt to speed up rspec and rubocop - can remove

File: app/controllers/nsm/claims_controller.rb [hunk 4]
```diff
@@ -1,10 +1,32 @@
...
-    def index
+    # open
+    # def index
+    #   @pagy, claims = pagy(Claim.pending_decision)
+    #   @claims = claims.map { |claim| BaseViewModel.build(:all_claims, claim) }
+    # end
+
+    def your
+      # @pagy, claims = pagy(Claim.pending_and_assigned_to(current_user))
+      # @claims = claims.map { |claim| BaseViewModel.build(:all_claims, claim) }
```
Comment (jsugarman) [inline app/controllers/nsm/claims_controller.rb:11]: remove

File: app/controllers/nsm/claims_controller.rb [hunk 5]
```diff
@@ -1,10 +1,32 @@
 module Nsm
   class ClaimsController < ApplicationController
-    def index
+    # open
+    # def index
+    #   @pagy, claims = pagy(Claim.pending_decision)
+    #   @claims = claims.map { |claim| BaseViewModel.build(:all_claims, claim) }
+    # end
```
Comment (jsugarman) [inline app/controllers/nsm/claims_controller.rb:7]: remove

**Review by patrick-laa — APPROVED** [review]

**Pull request merged**